### PR TITLE
Add New In tag for exclude_data_from_backup

### PR DESCRIPTION
**Pull request by nickvigilante**

### CODE REVIEW

File: _includes/v22.1/misc/table-storage-parameters.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 | Parameter name      | Description | Data type | Default value |
 |---------------------+----------------------+-----+------|
-| `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
+| **New in v22.1:** `exclude_data_from_backup` | Excludes the data in this table from any future backups. | Boolean | `false` |
```
Comment (stbof) [inline _includes/v22.1/misc/table-storage-parameters.md:3]:
```suggestion
| `exclude_data_from_backup` | **New in v22.1:** Excludes the data in this table from any future backups. | Boolean | `false` |
```

Comment (stbof) [reply _includes/v22.1/misc/table-storage-parameters.md:3]: I've been putting them in the description. Otherwise the table format looks bad.

**Review by stbof — APPROVED** [review]

**Comment by stbof** [comment]
See suggestion.

File: _includes/v22.1/misc/table-storage-parameters.md (hunk 1 above)
Comment (nickvigilante) [reply _includes/v22.1/misc/table-storage-parameters.md:3]: Ah, makes sense.

**Pull request merged**